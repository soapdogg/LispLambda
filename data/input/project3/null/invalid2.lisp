(NULL 23 23 T)