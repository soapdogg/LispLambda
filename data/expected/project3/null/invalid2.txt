Error! Expected length of NULL list is 2!    Actual: 4